sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/catalog/SessionCatalog.scala

```diff
@@ -48,7 +48,7 @@ import org.apache.spark.sql.util.{CaseInsensitiveStringMap, PartitioningUtils}
 import org.apache.spark.util.Utils
 
 object SessionCatalog {
-  val DEFAULT_DATABASE = "default"
+  val DEFAULT_DATABASE = SQLConf.get.defaultDatabase
```

Comment:
This makes it look like it's a runtime config. Let's write `getConf(StaticSQLConf.CATALOG_DEFAULT_DATABASE)`

Reply:
BTW it's a bit tricky to access the active SQLConf in scala object. Can we read the conf in `BaseSessionStateBuilder` and pass it to `SessionCatalog`?